File: railway-backend/src/models/governmentModels.ts
export interface GovScheme {
  id: string;
  title: string;
  description: string;
  category: string;
  eligibility: string;
  officialSource: string;
  officialUrl: string;
}

export interface WelfareBenefit {
  id: string;
  name: string;
  description: string;
  eligibility: string;
  requiredDocuments: string[];
  howToApply: string;
  officialSource: string;
}

export interface GovNotice {
  id: string;
  title: string;
  summary: string;
  date: string;
  authority: string;
  sourceUrl: string;
}

export interface GovHelpline {
  id: string;
  name: string;
  number: string;
  purpose: string;
  category: string;
}

File: app/src/main/java/com/example/smartcityassistant/data/government/GovernmentModels.kt
package com.example.smartcityassistant.data.government

data class GovScheme(
    val id: String,
    val title: String,
    val description: String,
    val category: String,
    val eligibility: String,
    val officialSource: String,
    val officialUrl: String
)

data class WelfareBenefit(
    val id: String,
    val name: String,
    val description: String,
    val eligibility: String,
    val requiredDocuments: List<String>,
    val howToApply: String,
    val officialSource: String
)

data class GovNotice(
    val id: String,
    val title: String,
    val summary: String,
    val date: String,
    val authority: String,
    val sourceUrl: String
)

data class GovHelpline(
    val id: String,
    val name: String,
    val number: String,
    val purpose: String,
    val category: String
)
